engine/src/main/java/org/hibernate/search/engine/metadata/impl/TypeMetadata.java

```diff
@@ -234,6 +240,20 @@ public BridgeDefinedField getBridgeDefinedFieldMetadataFor(String fieldName) {
 		return null;
 	}
 
+	public FacetMetadata getFacetMetadataFor(String facetFieldName) {
+		FacetMetadata result = facetFieldNameToFacetMetadata.get( facetFieldName );
+		if ( result != null ) {
+			return result;
+		}
+		for ( EmbeddedTypeMetadata element : embeddedTypeMetadata ) {
```

Comment:
For other maps there is, as we sometime want to include only non-embedded metadata (especially when generating the ES mapping). Here, we never want, but we might as well be consistent.
Anyway, we'll all agree that something has to change in TypeMetadata. Nothing urgent, though: I'd say it's 6.0 stuff.